packages/web/src/ee/features/chat/components/chatThread/mermaidDiagram.tsx

```
@@ -0,0 +1,626 @@
+        // In fullscreen, "reset" returns to the fitted view rather than 1:1.
+        if (fill && fitToSurface()) {
+            return;
+        }
+        setZoom(1);
+        setOffset({ x: 0, y: 0 });
+    }, [fill, fitToSurface]);
+
+    // Auto fit-to-surface once the fullscreen viewport has a measurable size
+    // and the SVG is in the DOM. We fit a single time (the user can pan/zoom
+    // freely afterwards, and "reset" re-fits on demand).
+    useEffect(() => {
```

Biome (2.5.0)

[error] This hook specifies more dependencies than necessary: svg.

(lint/correctness/useExhaustiveDependencies)